docs/_includes/src-highlightjs.adoc

```diff
@@ -0,0 +1,22 @@
+////
+Included in:
+
+- user-manual: Source Code Syntax Highlighting: HighlightJs installation
+
+:highlightjs-lang: https://highlightjs.org/download/
+////
+
+{highlightjs-org}[HighlightJS] is a popular client-side syntax highlighter that supports a broad range of {highlightjs-lang}[languages].
+
+To use highlightjs just add `:source-highlighter: highlightjs` in your AsciiDoc file.
```

Comment:
Core supports both "highlightjs" and "highlight.js". It's probably best to only recommend one. Which one should we recommend? (and thus stick with?)